examples/simple-sampler/stdout_exporter.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "opentelemetry/sdk/trace/exporter.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+#include <iostream>
+
+namespace trace    = opentelemetry::trace;
+namespace nostd    = opentelemetry::nostd;
+namespace sdktrace = opentelemetry::sdk::trace;
+
+class StdoutExporter final : public sdktrace::SpanExporter
+{
+  std::unique_ptr<sdktrace::Recordable> MakeRecordable() noexcept
+  {
+    return std::unique_ptr<sdktrace::Recordable>(new sdktrace::SpanData);
+  }
+
+  sdktrace::ExportResult Export(
```

Comment:
Typically implementations should be in a .cc file (unless they're templated classes/functions).